content/docs/setup/kubernetes/install/multicluster/shared-gateways/index.md

```diff
@@ -1,19 +1,22 @@
 ---
-title: Cluster-Aware Service Routing
+title: Shared control plane (multi-network)
 description: Leveraging Istio's Split-horizon EDS to create a multicluster mesh.
+description: Install an Istio mesh across multiple Kubernetes clusters using a shared control plane for diconnected cluster networks.
 weight: 85
 keywords: [kubernetes,multicluster]
 aliases:
     - /docs/examples/multicluster/split-horizon-eds/
+    - /docs/tasks/multicluster/split-horizon-eds/
 ---
 
-This example shows how to configure a multicluster mesh with a
-[single control plane topology](/docs/concepts/multicluster-deployments/#single-control-plane-topology)
-and using Istio's _Split-horizon EDS (Endpoints Discovery Service)_ feature (introduced in Istio 1.1) to
+Follow this guide to configure a multicluster mesh with a
+[shared control plane topology](/docs/concepts/multicluster-deployments/#shared-control-plane-topology)
+and using Istio's _Split-horizon EDS (Endpoints Discovery Service)_ feature to
 route service requests to other clusters via their ingress gateway.
-Split-horizon EDS enables Istio to route requests to different endpoints, depending on the location of the request source.
+Split-horizon EDS enables Istio to route requests to different endpoints,
```

Comment:
The same comment as https://github.com/istio/istio.io/pull/4638/files#r300825440